We can't run the contrib/sepgsql regression tests during build, because they
assume that some root-privilege-requiring hacks have been made to the SELinux
configuration.  So just dike out the settings that would cause them to be run.
Something similar to this will probably be done upstream for PG 9.1.2 and
later.


diff -Naur postgresql-9.1.1.orig/contrib/sepgsql/Makefile postgresql-9.1.1/contrib/sepgsql/Makefile
--- postgresql-9.1.1.orig/contrib/sepgsql/Makefile	2011-09-22 17:57:57.000000000 -0400
+++ postgresql-9.1.1/contrib/sepgsql/Makefile	2011-09-26 16:47:53.772278112 -0400
@@ -5,9 +5,6 @@
 	schema.o relation.o proc.o
 DATA_built = sepgsql.sql
 
-REGRESS = label dml misc
-REGRESS_OPTS = --launcher $(top_builddir)/contrib/sepgsql/launcher
-
 EXTRA_CLEAN = -r tmp *.pp sepgsql-regtest.if sepgsql-regtest.fc
 
 ifdef USE_PGXS
